src/GraphQL/Validation/ValidationContext.cs

```diff
@@ -191,16 +190,6 @@ public List<FragmentDefinition> GetRecursivelyReferencedFragments(Operation oper
             return fragments;
         }
 
-        /// <summary>
-        /// Returns a string representation of the specified node.
-        /// </summary>
-        public string Print(INode node) => AstPrinter.Print(node);
-
-        /// <summary>
-        /// Returns the name of the specified graph type.
-        /// </summary>
-        public string Print(IGraphType type) => SchemaPrinter.ResolveName(type);
-
```

Comment:
I was planning to see if we could decouple one or more of the AstPrinter / SchemaPrinter / SchemaBuilder libraries.  If that happens, these methods might need to move back here.  So I'd probably not remove them, but ok I guess....

Reply:
These methods created unnecessary dependencies.